### ISSUE TITLE
Assign service account to teams and/or workers

### ISSUE THREAD

**Issue by bonzofenix** [concourse/concourse-chart#239]
When I run tasks as a member of a team, it would be useful to have a service account associated to an aws role when deploying on EKS. This way I can rely on IAM for the permissions I want certain teams or a concourse cluster to have for aws. 
Maybe I am missing something but I do not see how could I implement this. 

I tried assigning an iam role to the workers service account but this does not propagate to the actual worker steps of a pipeline.

**Comment by taylorsilva** [comment]
Correct, I don't think there's currently anyway to achieve this. Concourse doesn't give you a way to share stuff from the workers environment with the containers it is creating on that worker.
